src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -172,9 +173,13 @@ private enum ResponseStatus
             authParams.RedirectUri = authenticationRequestParameters.RedirectUri.ToString();
 
             //MSA-PT
-            if (isMsaPassthrough)
+            if (brokerOptions.MsaPassthrough)
                 authParams.Properties[NativeInteropMsalRequestType] = ConsumersPassthroughRequest;
 
+            //WAM Header Title
+            if (!string.IsNullOrEmpty(brokerOptions.HeaderText))
```

Comment:
Assuming you have tested silent flow and this is still ok.

Reply:
Yes silent flow using broker (defaultOS account) works as expected